modules/reference/pages/configuration/config-channels.adoc

```diff
@@ -1,3 +1,134 @@
+[[ref.webui.config.channels]]
 = Configuration Channels
 
+As mentioned above, {productname} manages both central and local configuration channels and files.
```

Comment:
```suggestion
{productname} manages both central and local configuration channels and files.
```
There's no guarantee that people have read the above, or that the content referenced will remain 'above'